storage/imxrt-flash/flashsrv.c

```diff
@@ -257,15 +257,27 @@ static int flashsrv_fsEraseSectorf0(struct _meterfs_devCtx_t *devCtx, off_t offs
 {
 	(void)devCtx;
 
-	return flashsrv_eraseSector(0, offs);
+	const unsigned int fID = 0;
+
+	/* meterfs: drop mutex between sector erases to prevent starving other threads */
+	mutexUnlock(flashsrv_common.flash_memories[fID].lock);
+	mutexLock(flashsrv_common.flash_memories[fID].lock);
+
+	return flashsrv_eraseSector(fID, offs);
 }
 
 
 static int flashsrv_fsEraseSectorf1(struct _meterfs_devCtx_t *devCtx, off_t offs)
 {
 	(void)devCtx;
 
-	return flashsrv_eraseSector(1, offs);
+	const unsigned int fID = 1;
```

Comment:
What does this intermediate variable do? If it was to avoid warning, then the warning is alas still there